log4j-core/src/test/java/org/apache/logging/log4j/core/GcFreeAsynchronousLoggingTest.java

```diff
@@ -18,6 +18,7 @@
 
 import org.apache.logging.log4j.categories.AsyncLoggers;
 import org.apache.logging.log4j.core.async.AsyncLoggerContextSelector;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.